test/selenium/selenium-machines-network-page.py

```diff
@@ -0,0 +1,52 @@
+from testlib_avocado.seleniumlib import clickable, text_in, invisible
+from testlib_avocado.machineslib import MachinesLib
+
+
+class MachinesNetworkPageTestSuite(MachinesLib):
+    """
+    :avocado: enable
+    :avocado: tags=machines
+    """
+
+    def testNetworkPageInfo(self):
+        net_cmd_active = int(self.machine.execute('sudo virsh net-list | awk \'NR>=3{if($0!="")print}\' | wc -l')) + int(
```

Comment:
net_cmd_active = int(self.machines.execute('sudo virsh net-list | wc -l')) - 3

Will give you the number of active networks, which is what you want if I understand correctly.

Reply:
The 'net_cmd_active' should get the active networks of root and non-root to compare with page. So the 'net_cmd_active' need the the sum of 'virsh net-list' with sudo and without 'sudo'.Then I use this result to compare with page to make sure the page has gotten all information from backend.

Reply:
Oh right - till my point remain that you don't need awk because you are only interested in the amount of lines. So just remove -3 for each of the command results instead of running awk which is a bit hard to read.

Reply:
OK, got it